src/Sylius/Bundle/AttributeBundle/Form/Type/AttributeValueType.php

```diff
@@ -69,6 +69,7 @@ public function __construct(
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
+            ->add('localeCode')
```

Comment:
Shouldn't it be
```php
->add('localeCode', LocaleChoiceType::class, [
    'constraints' => [
        new NotBlank(['groups' => ['sylius']]),
    ],
])
```
And then we will use a transformer:
```php
$builder->get('localeCode')->addModelTransformer(
    new ReversedTransformer(new ResourceToIdentifierTransformer($this->localeRepository, 'code'))
);
```
With this solution we will be sure, that locale exist in the repository